crates/starknet-devnet-core/src/starknet/predeployed.rs

```diff
@@ -51,10 +51,12 @@ pub(crate) fn initialize_erc20_at_address(
         // necessary to set - otherwise minting txs cannot be executed
         ("Ownable_owner", Felt::from_prefixed_hex_str(CHARGEABLE_ACCOUNT_ADDRESS)?),
     ] {
-        let storage_var_address = get_storage_var_address(storage_var_name, &[])?.try_into()?;
+        let storage_var_address =
+            PatriciaKey::try_from(get_storage_var_address(storage_var_name, &[]).unwrap()).unwrap();
```

Comment:
More unwrapping.

Reply:
please could you give me some advice @FabijanC ?...i don't know what else to do to deal with these unwraps. Thanks in advance.

Reply:
please take your time, learn a bit, research and try! For example, one pointer:
![image](https://github.com/0xSpaceShard/starknet-devnet-rs/assets/2848732/d42633a4-5039-4b75-826d-648749531659)

Reply:
great thanks

Reply:
Hi @ivpavici , no i don't know how to handle this issue.